src/main/java/org/opensha/sha/earthquake/faultSysSolution/ruptures/util/SegmentationCalculator.java

```diff
@@ -1407,8 +1408,8 @@ public static void main(String[] args) throws IOException, DocumentException {
 		ClusterConnectionStrategy connStrat = new InputJumpsOrDistClusterConnectionStrategy(rupSet.getFaultSectionDataList(), distAzCalc, jumpDist, new ArrayList<>());
 		RuptureConnectionSearch rsConnSearch = new RuptureConnectionSearch(rupSet, distAzCalc,
 				1000d, RuptureConnectionSearch.CUMULATIVE_JUMPS_DEFAULT);
-		rupSet.buildClusterRups(rsConnSearch);
-		List<ClusterRupture> rups = rupSet.getClusterRuptures();
+		rupSet.addModule(ClusterRuptures.instance(rupSet, rsConnSearch));
```

Comment:
@kevinmilner I think this is potentially problematic, but first my assumptions:

 - that line 1411-new ==  line 1410-old >>  to build the Cluster Ruptures list using some helper class
 - line 1412-new & line 1411-old copy these objects to the local member variable 'rups'

My key concern is that it's reasonably easy to see what is happening in the old API from the method names and introspection in IDE etc. This is a  good thing & for new users especially, as the domain is being 'explained' through the API.  However, in the new API it less clear, as we seem to rely on the module class names and the module 'concept'. It' almost like we're creating a 'language' with only two or three verbs, which I fear may be rather difficult to understand for casual/new users.